src/docs/reference/modules/Contents/README.md

```diff
@@ -129,6 +129,136 @@ Well known properties include
     To log shapes call `@Orchard.ConsoleLog(Model.Content as object)` after calling `@await DisplayAsync(Model.Content)`
     This will allow the shape to execute, and populate the alternates for any child shapes.
 
+## Contents Module RESTful Web API
+
+The `OrchardCore.Contents` module provides RESTful API endpoints via [`ApiController`](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs) featuring endpoints to manage _content items_. These endpoints allow for operations such as retrieving, creating, updating, and deleting single content item instances. Access to these endpoints requires authentication and appropriate user role permissions.
+
+### Useful modules and libraries
+
+1. We suggest to use the docs about the [GraphQL module](https://docs.orchardcore.net/en/main/docs/reference/modules/Apis.GraphQL/), to be used for querying content items.
```

Comment:
Use relative links for all https://docs.orchardcore.net ones instead (see other pages for examples).

Reply:
Done in https://github.com/OrchardCMS/OrchardCore/pull/15512/commits/1d1fc8210f426595edb2bd3c11bf2ee862ca9f88

Reply:
Please make sure that the documentation build passes as well. Currently it fails, see the red X at the bottom of the PR.